app/js/components/Login.jsx

```diff
@@ -6,37 +6,73 @@
  * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
  * graphic logo is a trademark of OpenMRS Inc.
  */
-import React from 'react';
+import React, { Component } from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import axios from 'axios';
+import axiosInstance from '../config/axiosInstance';
```

Comment:
Since you are using the `axionInstance` to make your API call, you don't need to import `axios` again. You should remove this line:
```
import axios from 'axios';
```

Reply:
Oh, I accidentally forgot that there. Thanks. Addressed